lib/blockchain.go

```diff
@@ -323,28 +324,19 @@ func NewPoWBlockNode(
 		CumWork:          cumWork,
 		Header:           header,
 		Status:           status,
-		// All blocks have a committed status in PoW.
-		CommittedStatus: COMMITTED,
 	}
 }
 
 // NewPoSBlockNode is a new helper function to create a block node
 // as we need to control the value of the CommittedStatus field.
-func NewPoSBlockNode(
-	parent *BlockNode,
-	hash *BlockHash,
-	height uint32,
-	header *MsgDeSoHeader,
-	status BlockStatus,
-	committedStatus CommittedBlockStatus) *BlockNode {
+func NewPoSBlockNode(parent *BlockNode, hash *BlockHash, height uint32, header *MsgDeSoHeader, status BlockStatus) *BlockNode {
```

Comment:
Do we need a separate `NewPoSBlockNode` function now? Seems like we can reuse the original `NewBlockNode` function that's currently just used for PoW now